metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -790,6 +798,59 @@ def get_dataset_properties() -> DatasetPropertiesClass:
                 tags=[],
             )
 
+        def get_s3_tags() -> Optional[GlobalTagsClass]:
+            bucket_name = s3_util.get_bucket_name(
+                table["StorageDescriptor"]["Location"]
+            )
+            tags_to_add = []
+            if self.source_config.use_s3_bucket_tags:
+                bucket_tags = self.s3_client.get_bucket_tagging(Bucket=bucket_name)
```

Comment:
Please, can you handle the case if no bucket tags? 
For me it throws an exception if it does not exist.

Reply:
On it.

Reply:
Updated.  Added a `logger.warn` for when there are no tags.  Not sure if it should be a warn or debug message.